airframe-sql/src/test/scala/wvlet/airframe/sql/analyzer/TypeResolverTest.scala

```diff
@@ -155,6 +178,14 @@ class TypeResolverTest extends AirSpec {
   }
 
   test("resolve select from values") {
+    /**
+     * TODO This pattern cannot be round-tripped yet due to insufficient AllColumns handling.
+     * [generated sql from resolved plan]
+     * SELECT
+     *   MultiColumn(List(Literal(1), Literal(2), Literal(3)),None,None) AS id,
+     *   MultiColumn(List(Literal('one'), Literal('two'), Literal('three')),None,None) AS name
+     * FROM ((VALUES (1, 'one'), (2, 'two'), (3, 'three'))) AS t(id, name)
+     */
     val p = analyze("SELECT * FROM (VALUES (1, 'one'), (2, 'two'), (3, 'three')) AS t (id, name)")
```

Comment:
This pattern will be covered in another PR as a part of `AllColumns` handling improvement.